wrangler-service/src/main/java/co/cask/wrangler/service/connections/ConnectionService.java

```diff
@@ -252,6 +276,11 @@ public void list(HttpServiceRequest request, HttpServiceResponder responder,
   public void delete(HttpServiceRequest request, HttpServiceResponder responder,
                      @PathParam("id") final String id) {
     try {
+      if (isDefaultConnection(id)) {
+        responder.sendError(HttpURLConnection.HTTP_BAD_REQUEST,
+                            String.format("Cannot delete default connection %s", id));
```

Comment:
default is an ambiguous term here. I think we can say 'Cannot delete admin controlled connection %s.'